src/main/src/main/java/org/geoserver/config/impl/ServiceInfoImpl.java

```diff
@@ -247,35 +257,26 @@ public void setVerbose(boolean verbose) {
     public int hashCode() {
         final int prime = 31;
         int result = 1;
-        result = prime * result + ((abstrct == null) ? 0 : abstrct.hashCode());
-        result = prime
-                * result
-                + ((accessConstraints == null) ? 0 : accessConstraints
-                        .hashCode());
+        result = prime * result + ((getAbstract() == null) ? 0 : getAbstract().hashCode());
```

Comment:
Confused, why is the commit changing all field access to getters? The method has visibility of the fields and I see no late/lazy inizialization of them.  This kind of change is likely going to be reverted next time someone uses a IDE code generator to update hashCode/equals